incubator/kitura/templates/default/Tests/ApplicationTests/KituraStackTests.swift

```diff
@@ -3,6 +3,11 @@ import Kitura
 import KituraNet
 import XCTest
 import AppsodyKitura
+#if swift(>=4.1)
```

Comment:
This is fine, but this stack is never going to be used with Swift 4.0, so we could simplify this by removing the version check.